### PR TITLE
BC-3755-Header update: logo, icons, paddings [NBC, Brandenburg. Mobile]

### PR DESCRIPTION
**Pull request by Michaellinaresxk**
# Description
Added css rules to remove icons on header for mobile UI consistency features and place status event in homepage for all instances.
Screen Shot 2023-07-12 at 11 53 49 AM

## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-3755

## Changes
Removed for mobile:
 1. fullscreen mode Icon, 
 2. QR-code Icon , 
 3. Question Icon for the mobile. 
 4. Breadcrumbs/page title inside the header.
 
-Display Logo of the platform 
-Place the status event on homepage in all instances

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

**Review by VikDavydiuk — COMMENTED** [review]
photo_2023-07-14_11-52-15
photo_2023-07-14_11-52-15 (2)
photo_2023-07-14_11-52-15 (3)
photo_2023-07-14_11-52-15 (4)
photo_2023-07-14_11-52-15 (5)
photo_2023-07-14_11-52-15 (6)
photo_2023-07-14_11-52-15 (7)
photo_2023-07-14_11-52-15 (8)
photo_2023-07-14_11-52-15 (9)
This is how it shows up on my phone (OnePlus Nord N10 5G)

File: config/global.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const util = require('util');
 const {
 	PUBLIC_BACKEND_URL,
 	SC_DOMAIN = 'localhost',
-	SC_THEME = 'default',
+	SC_THEME = 'n21',
```
Comment (sszafGCA) [inline config/global.js:6]: do not change theme

File: gulpfile.js [hunk 2]
```diff
@@ -51,7 +51,7 @@ const baseStyles = [
 ];
 
 function themeName() {
-	return process.env.SC_THEME || 'default';
+	return process.env.SC_THEME || 'n21';
```
Comment (sszafGCA) [inline gulpfile.js:54]: Same here

**Comment by sszafGCA** [comment]
As BC-3743 is done as a part of this story I left the comments on some scss files to do refactor of code. Please look at those comment:
//TODO in BC-3743

File: static/styles/lib/alerts.scss [hunk 3]
```diff
@@ -161,11 +162,10 @@
 			}
 
 			@include media-breakpoint-down(sm) {
-				left: -216px;
-
+				left: -290px;
 				&::after,
 				&::before {
-					left: 210px;
+					left: 285px;
```
Comment (VikDavydiuk) [inline static/styles/lib/alerts.scss:168]: You use rem, em, px. Can we use one of them? Also, avoid magic numbers

File: static/styles/lib/alerts.scss [hunk 4]
```diff
@@ -65,6 +65,7 @@
 			border: none;
 			margin-left: -0.75rem;
 			background: transparent;
+			margin-top: -0.5em;
```
Comment (MartinSchuhmacher) [inline static/styles/lib/alerts.scss:68]: can we avoid using negative values at all here?

File: static/styles/lib/alerts.scss [hunk 5]
```diff
@@ -161,11 +162,10 @@
 			}
 
 			@include media-breakpoint-down(sm) {
-				left: -216px;
-
+				left: -290px;
 				&::after,
 				&::before {
-					left: 210px;
+					left: 17.8em;
```
Comment (MartinSchuhmacher) [inline static/styles/lib/alerts.scss:168]: why overwriting a fixed px value with em value? makes it hard to find issues later

File: static/styles/lib/header.scss [hunk 6]
```diff
@@ -39,8 +39,17 @@
...
+	@include media-breakpoint-down(md) {
+		height: 56px;
+  }
+
+	@include media-breakpoint-down(sm) {
+		height: 56px;
+  }
 	.minimal-button {
 		float: right;
+		margin-top: 3px;
```
Comment (MartinSchuhmacher) [inline static/styles/lib/header.scss:52]: same here, as the more specificated values follow rem values, I would not like to have different units as base

File: static/styles/lib/loggedin.scss [hunk 7]
```diff
@@ -814,3 +814,66 @@ footer {
 .school-logo {
 	max-height: 60px;
 }
+
+.content-wrapper .cloud-logo {
+	display: inline-block;
+	vertical-align: top;
+	width: 50px;
+	height: 30px;
+	margin: -5px 0.25rem 0 0;
```
Comment (MartinSchuhmacher) [inline static/styles/lib/loggedin.scss:823]: dont mix px and rem within one element

File: static/styles/lib/loggedin.scss [hunk 8]
```diff
@@ -814,3 +814,66 @@ footer {
...
+  .breadcrumb,
+  .btn-fullscreen,
+  .qr-code-action,
+  .help-drodown {
+    display: none !important;
+  }
+
+	.content-wrapper .cloud-logo {
+		width: 9.375em;  /* (150px / 16px) */
+		height: 50px;
```
Comment (MartinSchuhmacher) [inline static/styles/lib/loggedin.scss:858]: see above about mixing

File: static/styles/lib/loggedin.scss [hunk 9]
```diff
@@ -814,3 +814,66 @@ footer {
...
+
+  .breadcrumb,
+  .btn-fullscreen,
+  .qr-code-action,
+  .help-drodown {
+    display: none !important;
+  }
+
+	.content-wrapper .cloud-logo {
+		width: 9.375em;  /* (150px / 16px) */
```
Comment (MartinSchuhmacher) [inline static/styles/lib/loggedin.scss:857]: is the calculation important for something? if it is, then put the origin values into variables and calculate the value here based on them. if not: remove the comment

File: theme/brb/style.scss [hunk 10]
```diff
@@ -59,11 +61,22 @@ body {
...
             @media (max-width: 991px) {
-                .cloud-logo {
-                    width: 100% !important;
-                    margin-left: 0 !important;
+               .cloud-logo {
+                    background: $logoBackgroundTablet;
+                    margin-left: 0.5em!important;
+                    position: absolute;
+                    left: 0;
+                    width: 13rem !important;
```
Comment (MartinSchuhmacher) [inline theme/brb/style.scss:69]: differentiate between parent and root element in styling one specific element does not sound like a good idea

File: theme/brb/style.scss [hunk 11]
```diff
@@ -59,11 +61,22 @@ body {
...
                 }
             }
+            @media (max-width: 557px) {
+                .cloud-logo {
+                    background: $logoBackgroundMobile !important;
+                    width: 7rem !important;
+                    height: 50px;
```
Comment (MartinSchuhmacher) [inline theme/brb/style.scss:76]: see above (mixing)

File: theme/default/style.scss [hunk 12]
```diff
@@ -28,6 +28,8 @@ aside.nav-sidebar{
             padding: 0;
             .cloud-logo{
                 background: $logoBackgroundTablet;
+                width: 9.375em;
+                height: 2rem;
```
Comment (MartinSchuhmacher) [inline theme/default/style.scss:32]: see above (parent vs root)

File: theme/n21/style.scss [hunk 13]
```diff
@@ -55,9 +55,13 @@ body {
 
             @media (max-width: 991px) {
                 .cloud-logo {
-                    width: 100% !important;
-                    margin-left: 0 !important;
-                }
+                    margin-left: 0.2em!important;
+                    background: $logoBackgroundMobile !important;
+                    width: 9.375em;
+                    height: 2rem;
```
Comment (MartinSchuhmacher) [inline theme/n21/style.scss:61]: see above

File: theme/thr/style.scss [hunk 14]
```diff
@@ -48,9 +48,13 @@ body {
 
             @media (max-width: 991px) {
                 .cloud-logo {
-                    width: 100% !important;
-                    margin-left: 0 !important;
-                }
+                    margin-left: 0.2em!important;
+                    background: $logoBackgroundMobile !important;
+                    width: 9.375em;
+                    height: 2rem;
```
Comment (MartinSchuhmacher) [inline theme/thr/style.scss:54]: see above

**Review by MartinSchuhmacher — CHANGES_REQUESTED** [review]

**Review by MartinSchuhmacher — APPROVED** [review]

**Review by VikDavydiuk — COMMENTED** [review]
image
image

**Review by VikDavydiuk — APPROVED** [review]

**Pull request merged**